### PR TITLE
#165058763 Add dynamic fixtures

### PR DESCRIPTION
**Pull request by olusoladavid**
#### What Does This PR Do?

- Adds a simple infrastructure for creating dynamic fixtures in the app (in development environments). Also implements a dynamic fixture for the Event resource. This will improve the developer experience when devs need to seed data whose visibility is time or resource dependent.

#### Description Of Task To Be Completed

- [x] Install dynamic fixtures app for Django
- [x] Setup dynamic fixture for event
- [x] Update README

#### Any Background Context You Want To Provide?
*Current Behaviour on Develop Branch*

All fixtures are currently static

#### How can this be manually tested?
- Setup the project with Docker as outlined in the README
- Run `make build`
- Start the docker server container using `docker start dev_andela_social_server`
- Activate shell in the container using `docker exec -it dev_andela_social_server bash`
- In bash, run `python manage.py load_dynamic_fixtures`
- You should get a success message and can also confirm in your database that the fixture has been loaded

#### What are the relevant pivotal tracker stories?
[#165058763](https://www.pivotaltracker.com/story/show/165058763)

#### Screenshot(s)
 - N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,19 +160,23 @@ Reference:
...
-to get the public URL. e.g. `https://05473cbc.ngrok.io` 
+```
+to get the public URL. e.g. `https://05473cbc.ngrok.io`
 
 ### UI MOCK
 
 The UI mock for the project is available [here](https://www.figma.com/file/Yn3JRZ3YLBVSg4o8L9dhIAv2/Andela_Socials)
 
-### DUMMY DATA
+### STATIC DUMMY DATA
```
Comment (brianbirir) [inline README.md]: @olusoladavid this section and dynamic dummy data section should be under one single section titled **Database Seeding** for example, and then added as subtitles.

Sample:

```
### Database Seeding
#### Static Dummy Data
#### Dynamic Dummy Data
```

**Review by brianbirir — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (olusoladavid) [reply README.md]: Yeah. Sounds better. Thanks for the suggestion. I will update the README with the suggested structure.

File: server/api/fixtures/001_create_events.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  def load(self):
+    eventObjects = []
+    creators = list(AndelaUserProfile.objects.all())
+    categories = list(Category.objects.all())
+    timezones = ['Africa/Lagos', 'Africa/Nairobi']
+    venues = ['Lagos', 'Nairobi']
+    fake = Faker()
+    event_start_date = datetime.today()
+    # generate 10 fixtures at a time
+    for _ in range(10):
```
Comment (brianbirir) [inline server/api/fixtures/001_create_events.py:18]: This works well @olusoladavid . Using `_` in the loop function is a smart move

**Comment by brianbirir** [comment]
@olusoladavid in general, this feature works as expected. It creates 10 events dynamically. Once you do the change on the `README.md` file, we can merge.

**Review by brianbirir — APPROVED** [review]

**Comment by brianbirir** [comment]
@olusoladavid I've seen the updated change on the README document. It looks good so you can now merge.

**Pull request merged**